src/index.ts

```diff
@@ -67,65 +63,84 @@ export function generateWords(nr: number, words: string[] = []): string {
     nr = Math.abs(nr);
   }
 
+  let remainder = 0;
+  let quotient: number;
```

Comment:
I had to google "quotient" - a result obtained by dividing one quantity by another. Well it makes sense but what does it represent because the actual division is Math.trunc(nr / divider), you are saying "divider"  but this actually represents the steps between tens and hundreds and thousands and millions and so on, not sure if "divider" makes you think that of that though the same as  "quotient".